src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version);
```

Comment:
I know that I might have said to do this and it is better then before I do have to agree with @eerhardt that it isn't all that helpful because it will always be 4.0.0.0. At this point however I also agree with @priya91 that I don't know if any good way to get a better version, in the context of .NET Native at least because reflection isn't going to work. 

@ericstj any other options?